client/web/src/search/home/AnnouncementBlurb.module.scss

```
@@ -0,0 +1,25 @@
+.saturn-gradient-border {
+    position: relative;
+    max-width: 800px;
+    text-align: center;
+}
+
+.saturn-gradient-border::after {
+    content: "";
+    position: absolute;
+    z-index: 1;
+    inset: 0;
+    border-radius: 5px;
+    border: 3px solid transparent;
+    background: var(--saturn-gradient);
+    -webkit-mask:
+       linear-gradient(#fff 0 0) padding-box, 
+       linear-gradient(#fff 0 0);
+    -webkit-mask-composite: xor;
+            mask-composite: exclude;
```

The browser prefixes are not needed here as we have a plugin that can add them automatically. Remove them, then test on all browsers (Chromium, Safari, Firefox) to verify behavior is consistent.
```suggestion
    mask:
       linear-gradient(#fff 0 0) padding-box, 
       linear-gradient(#fff 0 0);
    mask-composite: exclude;
```